src/MyEventEmitter.js

```diff
@@ -1,12 +1,64 @@
-'use strict';
-
 export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this._events = {};
+  }
+
+  on(name, listener) {
+    if (!this._events[name]) {
+      this._events[name] = [];
+    }
+
+    this._events[name].push(listener);
```

Comment:
let's reduce `this.` across the code
```suggestion
		const { events } = this;
    if (!events[name]) {
      events[name] = [];
    }

    events[name].push(listener);
```